### PR TITLE
Adding Resources list to Inclusive Events Chapter

### PR DESCRIPTION
**Pull request by BrainonSilicon**
### Summary

This PR merges a draft of the Resources page for the Inclusive Events chapter discussed in #3307 into a WIP branch of the chapter. 

### List of changes proposed in this PR (pull-request)

* Added resources 
* Organized resources into sections

### What should a reviewer concentrate their feedback on?

A review should focus on what a "first draft" of this content should look like
- [ ] Are there any additional categories that should be added for the first draft of this material?
- [ ] Should the formatting of the headings and links change?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.

### CODE REVIEW

File: book/website/collaboration/inclusive-events/inclusive-events-resources.md [hunk 1]
```diff
@@ -20,5 +21,92 @@
 (cl-inclusive-events-resources-reading)=
 ## Further Reading
 
+* [The 7 Principles of Universal Design](https://universaldesign.ie/about-universal-design/the-7-principles)
+* (Book) [The Art of Gathering: How We Meet and Why It Matters](https://www.priyaparker.com/book-art-of-gathering) by Priya Parker
+
+(cl-inclusive-events-resources-other)=
+## Additional Resources 
+
+This chapter cannot (and should not) replace working with members of your specific community to plan and host events.
```
Comment (gedankenstuecke) [inline book/website/collaboration/inclusive-events/inclusive-events-resources.md:30]:
```suggestion
These resources cannot (and should not) replace working with members of your specific community to plan and host events. 
```

Of course also true for the chapter at large, but I guess at this particular space mostly about the resources linked below?

File: book/website/collaboration/inclusive-events/inclusive-events-resources.md [hunk 2]
```diff
@@ -20,5 +21,92 @@
...
+* Citizen's Advice: [How to run focus groups](https://www.citizensadvice.org.uk/Global/CitizensAdvice/Equalities/How%20to%20run%20focus%20groups%20guide.pdf)
+* [Guidelines for Conducting a Focus Group](https://irep.olemiss.edu/wp-content/uploads/sites/98/2016/05/Trinity_Duke_How_to_Conduct_a_Focus_Group.pdf)
+
+### Community-Specific Accessibility Resources
+
+#### Deaf and Hard of Hearing 
+* [Sign Solutions: How to plan a conference with event interpretation for Deaf people](https://www.signsolutions.uk.com/how-to-plan-a-conference-with-event-interpretation-for-deaf-people/)
+* [Royal National Institute for Deaf People: Make your meetings deaf aware](https://rnid.org.uk/information-and-support/deaf-awareness/make-your-meetings-deaf-aware/)
+* (UK only) [Action Deafness Interpreting: British Sign Language Interpreter Services](https://actiondeafness.org.uk/services/interpreting/)
+* [Working with sign language interpreters for events](https://www.signsolutions.uk.com/working-with-sign-language-interpreters-for-events/)
```
Comment (gedankenstuecke) [inline book/website/collaboration/inclusive-events/inclusive-events-resources.md:59]: I would potentially move the community specific resources towards the end, to move from the general to the more specific one.

File: book/website/collaboration/inclusive-events/inclusive-events-resources.md [hunk 3]
```diff
@@ -20,5 +21,92 @@
...
+* [University College London's Tips / Checklist for making events accessible](https://www.ucl.ac.uk/equality-diversity-inclusion/equality-areas/disability-equality/tips-checklist-making-events-accessible)
+* [University of British Columbia Accessible and Inclusive Event Planning](https://equity.ubc.ca/resources/accessible-and-inclusive-event-planning/)
+* [University of York: Inclusive and Accessible Events: Good Practice Guidance](https://www.york.ac.uk/media/abouttheuniversity/equality/documents/Inclusive-and-accessible-events-good-practice-guidelines.pdf)
+* Equality and Human Rights Commission: [Engaging with Disabled People An Events Planning Guide](https://www.equalityhumanrights.com/sites/default/files/housing-and-disabled-people-engaging-with-disabled-people-event-planning-guide.pdf)
+* [Independent Living Institute: Holding Meetings](https://www.independentliving.org/docs2/daakit45.html)
+* [Making in-person events accessible](https://communications.admin.ox.ac.uk/guidance-for-planning-and-organising-accessible-events#collapse2055661)
+
+### Planning Inclusive Events
```
Comment (gedankenstuecke) [inline book/website/collaboration/inclusive-events/inclusive-events-resources.md:76]: I haven't checked, but are those also listed in the more general event organisation chapters? If so, would it be worth linking out to the broader resources for inclusive event organisation within TTW (please ignore if i misremember and that doesn't actually exist 🙈 )

**Review by gedankenstuecke — APPROVED** [review]
This looks like a good overview overall, just left 2-3 suggestions/comments below. Generally feels like it's at a stage where it could be merged into the larger WIP-branch to have a complete WIP version!

File: book/website/collaboration/inclusive-events/inclusive-events-resources.md (hunk 3 above)
Comment (BrainonSilicon) [reply book/website/collaboration/inclusive-events/inclusive-events-resources.md:76]: That actually aren't (but possibly could be!)

**Comment by BrainonSilicon** [comment]
> This looks like a good overview overall, just left 2-3 suggestions/comments below. Generally feels like it's at a stage where it could be merged into the larger WIP-branch to have a complete WIP version!

Awesome! 

I made those changes and so i think we're all good to merge and collate things into one place ahead of the re-write

**Pull request merged**